articles/azure-maps/map-add-pin.md

```diff
@@ -52,38 +51,133 @@ There are four different types of point data that can be added to the map:
+- `atlas.Shape` class is similar to the GeoJSON feature. Both consist of a GeoJSON geometry and a set of properties that contain metadata associated to the geometry. If a GeoJSON object is added to a data source, it can easily be rendered in a layer. However, if the coordinates property of that GeoJSON object is updated, the data source and map don't change. That's because there's no mechanism in the JSON object to trigger an update. The shape class provides functions for updating the data it contains. When a change is made, the data source and map are automatically notified and updated.
 
 The following code sample creates a GeoJSON Point geometry and passes it into the `atlas.Shape` class to make it easy to update. The center of the map is initially used to render a symbol. A click event is added to the map such that when it fires, the coordinates of the mouse are used with the shapes `setCoordinates` function. The mouse coordinates are recorded at the time of the click event. Then, the `setCoordinates` updates the location of the symbol on the map.
 
-<br/>
+```javascript
+function InitMap()
+{
+    var map = new atlas.Map('myMap', {
+        center: [-122.33, 47.64],
+        zoom: 13,
+        view: "Auto",
+
+        //Add authentication details for connecting to Azure Maps.
+        authOptions: {
+            authType: 'subscriptionKey',
+            subscriptionKey: '{Your-Azure-Maps-Subscription-key}'
+        }
+    });
+
+    //Wait until the map resources are ready.
+    map.events.add('ready', function () {
+
+      /*Create a data source and add it to the map*/
+      var dataSource = new atlas.source.DataSource();
+      map.sources.add(dataSource);
+      var point = new atlas.Shape(new atlas.data.Point([-122.33, 47.64]));
+      //Add the symbol to the data source.
+      dataSource.add([point]);
+
+      /* Gets co-ordinates of clicked location*/
+      map.events.add('click', function(e){
+        /* Update the position of the point feature to where the user clicked on the map. */
+        point.setCoordinates(e.position);
+      });
+
+      //Create a symbol layer using the data source and add it to the map
+      map.layers.add(new atlas.layer.SymbolLayer(dataSource, null));
+    });
+}
+
+```
+
+:::image type="content" source="./media/map-add-symbol/add-symbol-layer.png" alt-text="A screenshot of map with a pin added using the symbol layer.":::
```

Comment:
The attribution is still an old one. Can you help to use the latest SDK version and update the screenshot? Thank you so much!